dug_4_osfv.md

```diff
@@ -0,0 +1,214 @@
+class: center, middle, intro
+
+# &#x1F386; Dasharo Open Source Firmware Validation &#x1F386;
+
+.center[<img src="/remark-templates/dasharo-presentation-template/images/dasharo-sygnet-white.svg" width="150px" style="margin-left:-20px">]
+
+???
+
+---
+
+# Agenda
+
+* Introduction to Dasharo Open Source Firmware Validation (OSFV)
+* Exploring OSFV v0.2
+* Recent improvements
+* Getting started with OSFV
+* Upcoming Features: OSFV v0.3 and Beyond
+* Q&A
+
+???
+
+---
+
+# History
+
+.center.image-99[![](/img/osfv.png)]
+
+* We use OSFV at least since 2018, when we started validate PC Engines releases
+* Using those scripts we over **50k** tests publicly releasing **150+** binaries
+  based on open-source firmware.
+* Initially it was closed source because of assumption that validation provides
+  majority of the value in open-source firmware development.
+
+---
+
+# Dasharo OSVF
+
+.center[https://github.com/Dasharo/open-source-firmware-validation]
+
+* Dasharo Open Sourve Firmware Validation consist of MIT-licensed scripts
+  written in Robot Frameworks with the purpose of validation of open-source
```

Comment:
```suggestion
  written in Robot Framework with the purpose of validation of open-source
```

Please consider adding one slide background for Robot Framework, it is huge topic and well recognized framework with its own conference as well as foundation.

Reply:
@pietrushnic Pushed 2nd version, added RF slide